modules/runtime/nuxeo-launcher-commons/src/main/java/org/nuxeo/launcher/config/ServerConfigurator.java

```diff
@@ -614,8 +617,11 @@ public InstanceInfo getInfo(String clid, List<LocalPackage> pkgs) {
             nxInstance.packages.add(info);
             pkgTemplates.addAll(info.templates);
         }
-        // templates
         nxInstance.config = new ConfigurationInfo();
+        // profiles
+        nxInstance.config.profiles.addAll(
+                Arrays.asList(generator.getEnvironment(NUXEO_PROFILES, "").split(TEMPLATE_SEPARATOR)));
```

Comment:
```suggestion
                List.of(generator.getEnvironment(NUXEO_PROFILES, "").split(TEMPLATE_SEPARATOR)));
```

Reply:
Not in this case, we get an array so it makes sense to use `Arrays` API.